packages/webapp/src/components/CardWithStatus/TaskCard/TaskCard.jsx

```diff
@@ -33,7 +33,9 @@ export const taskStatusTranslateKey = {
 };
 
 const getDate = (date, language = 'en') => {
-  return new Intl.DateTimeFormat(language, { dateStyle: 'medium' }).format(new Date(date));
+  return new Intl.DateTimeFormat(language, { timeZone: 'UTC', dateStyle: 'medium' }).format(
```

Comment:
@Tbrid just wondering about the addition of the timeZone here - I am not sure we should be setting the timeZone